python/tests/res/enkf/test_everest.py

```diff
@@ -0,0 +1,90 @@
+import os
+import sys
+from ecl.test import ExtendedTestCase, TestAreaContext
+from ecl.util import BoolVector
+
+from res.test import ErtTestContext
+from res.enkf import EnkfFs, EnkfConfigNode, NodeId, EnkfNode
+from res.enkf import EnKFMain, ErtRunContext
+from res.enkf.enums import EnKFFSType
+from res.enkf.enums import RealizationStateEnum
+
+
+class EverestTest(ExtendedTestCase):
```

Comment:
Can we please rename this test? There is no reason that _libres_ should have the slightest clue what _Everest_ is, and the same goes mentally for us developers when doing work in _libres_. Could we find a name that is descriptive of the new functionality tested instead?

Reply:
Done